packages/components/theme.ts

```diff
@@ -8,49 +8,32 @@ export const theme = {
         heading: 'IBM Plex Sans, sans-serif',
     },
     buttons: {
-        uno: {
+        primary: {
             cursor: 'pointer',
-            bg: 'gray.3',
-            color: 'gray.8',
-            borderRadius: 'sm',
+            bg: 'rgba(16, 16, 16, 1)',
+            color: '#fff',
+            borderRadius: '36px',
+            paddingTop: '2px',
+            paddingBottom: '2px',
             px: 4,
-            ':hover': {
-                bg: 'gray.7',
-                color: 'gray.3',
-            },
```

Comment:
And it would be nice if you add a simple hover effect. Just like lower the opacity on hover or anything that show we are hovering the button